### PR TITLE
Update TRG 4.06 - Notice for docker images (old draft version)

### PR DESCRIPTION
**Pull request by Siegfriedk**
Please be aware: This TRG 4.06 is in draft for a long time and was not updated.

This PR is here to clarify with the community IF we delete the TRG 4.6 draft OR finish it.

### CODE REVIEW

**Comment by FaGru3n** [comment]
IMHO we should finish it.

**Comment by evegufy** [comment]
> IMHO we should finish it.

not sure. the point of the update was [adding the hashicorp vault image](https://github.com/SSIRKC/eclipse-tractusx.github.io/commit/4315da62d9d2f01ded3419aae51c4c58ee371417#diff-6c4b30e2734b6b1ea2d21e9d4b8cac47550893b85a077e94fb1f221cdd792419), @tom-rm-meyer-ISST right? do you still see that change as necessary?

**Comment by tom-rm-meyer-ISST** [comment]
> > IMHO we should finish it.
> 
> not sure. the point of the update was [adding the hashicorp vault image](https://github.com/SSIRKC/eclipse-tractusx.github.io/commit/4315da62d9d2f01ded3419aae51c4c58ee371417#diff-6c4b30e2734b6b1ea2d21e9d4b8cac47550893b85a077e94fb1f221cdd792419), @tom-rm-meyer-ISST right? do you still see that change as necessary?

No it's not. We clarified that this is not needed for us, as we only use the vault for local testing and it's not part of our images. Thus, we should remove it

File: docs/release/trg-4/trg-4-06.md [hunk 1]
```diff
@@ -148,9 +131,9 @@ You have to adapt some of the provided links to match your used version.
 - GitHub project: [https://github.com/dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker)
 - DockerHub: [https://hub.docker.com/_/microsoft-dotnet-aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet)
 
-### Linux
+### Hashicorp Vault image
```
Comment (tom-rm-meyer-ISST) [inline docs/release/trg-4/trg-4-06.md:134]: I added this, but at least I don't need it anymore as base images are just relevant for distribution purposes and NOT for local testing purposes.

AFAIK the EDC team once distributed images with a vault. But likely not as a base image. @paullatzelsperger or @wolf4ood can you confirm?

File: docs/release/trg-4/trg-4-06.md [hunk 2]
```diff
@@ -68,50 +81,20 @@ Closing the notice, we provide a general statement about potentially contained o
...
 The following examples are shown as reference, to see already existing and complete versions of a 'Notice for docker images'.
 They **can not** be used for your product without modifications.
 
-- [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
-- [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk/blob/main/DOCKER_NOTICE.md)
+Good example for notice integrated in toplevel `README.md`: [IRS](https://github.com/eclipse-tractusx/item-relationship-service#notice-for-docker-image)
+
+Good example for a dedicated notice file: [edc-controlplane-memory-hashicorp-vault](https://github.com/eclipse-tractusx/tractusx-edc/edit/main/edc-controlplane/edc-controlplane-memory-hashicorp-vault/notice.md)
 
 ## Already collected base image information
```
Comment (tom-rm-meyer-ISST) [inline docs/release/trg-4/trg-4-06.md:100]: Something that helped me was the information that the image later on running in the container is the base image under consideration, e.g. meaning that in case of multi-stage builds, the last one's the relevant. This would be a quality of live improvement. Opinions?

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
We need to clarify the vault. Also I think we should add an understanding of "base image" as I also wasn't aware of that.

File: docs/release/trg-4/trg-4-06.md (hunk 2 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:100]: You mean in a sense of adding a hint to the TRG?

File: docs/release/trg-4/trg-4-06.md (hunk 1 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:134]: In this case its just a mentioning of possibility. But i can remove it.

File: docs/release/trg-4/trg-4-06.md [hunk 3]
```diff
@@ -148,9 +131,9 @@ You have to adapt some of the provided links to match your used version.
...
+### Hashicorp Vault image
 
-- Base image reference (example): `alpine:3.17`
-- Dockerfile: [alpinelinux/docker-alpine:3.17](https://github.com/alpinelinux/docker-alpine/blob/681b8c677aaed66e48a5ce721509647bd4dcd017/x86_64/Dockerfile)
-- GitHub project: [https://github.com/alpinelinux/docker-alpine](https://github.com/alpinelinux/docker-alpine)
-- DockerHub: [https://hub.docker.com/_/alpine](https://hub.docker.com/_/alpine)
+- Base image reference (example): `vault:13.3`
+- Dockerfile (:13.3): [vault:13.3](https://github.com/hashicorp/docker-vault/blob/v1.13.3/0.X/Dockerfile)
+- GitHub project: [https://github.com/hashicorp/docker-vault](https://github.com/hashicorp/docker-vault)
+- DockerHub: [hhttps://hub.docker.com/_/vault](https://hub.docker.com/_/vault)
```
Comment (evegufy) [inline docs/release/trg-4/trg-4-06.md:139]: Please revert the change of replacing the linux alpine base image which the vault image

File: docs/release/trg-4/trg-4-06.md (hunk 1 above)
Comment (evegufy) [reply docs/release/trg-4/trg-4-06.md:134]: I think the change regarding the vault image should be revert (and for sure not replace the linux alpine image) as  I assume the image @tom-rm-meyer-ISST is referring to also uses temurin as base: https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-dataplane/edc-dataplane-hashicorp-vault/src/main/docker/Dockerfile#L23

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/release/trg-4/trg-4-06.md (hunk 3 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:139]: That has to be a diff / move conflict. Will fix it

File: docs/release/trg-4/trg-4-06.md (hunk 1 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:134]: @evegufy i'm not sure; what you reference has nothing to do with vault besides the name of it. But yes @tom-rm-meyer-ISST whats your status with the vault usage?

File: docs/release/trg-4/trg-4-06.md (hunk 3 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:146]: fixed

File: docs/release/trg-4/trg-4-06.md (hunk 2 above)
Comment (tom-rm-meyer-ISST) [reply docs/release/trg-4/trg-4-06.md:100]: Exactly, just for better understandability for new joiners

Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md:98]: Hint added to the Why section

**Comment by Siegfriedk** [comment]
> > > IMHO we should finish it.
> > 
> > 
> > not sure. the point of the update was [adding the hashicorp vault image](https://github.com/SSIRKC/eclipse-tractusx.github.io/commit/4315da62d9d2f01ded3419aae51c4c58ee371417#diff-6c4b30e2734b6b1ea2d21e9d4b8cac47550893b85a077e94fb1f221cdd792419), @tom-rm-meyer-ISST right? do you still see that change as necessary?
> 
> No it's not. We clarified that this is not needed for us, as we only use the vault for local testing and it's not part of our images. Thus, we can remove it from my point of view.

removed

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM

**Review by evegufy — APPROVED** [review]

File: docs/release/trg-4/trg-4-06.md (hunk 1 above)
Comment (Siegfriedk) [reply docs/release/trg-4/trg-4-06.md]: @tom-rm-meyer-ISST vault is gone
